artifactory.py

```diff
@@ -1579,6 +1579,59 @@ def find_permission_target(self, name):
             return obj
         return None
 
+    def get_users(self):
+        request_url = self.drive + "/api/security/users"
+        r = self.session.get(request_url, auth=self.auth)
+        r.raise_for_status()
+        response = r.json()
+        users = []
+        for i in response:
+            user = User(self, i["name"])
+            user.read()
```

Comment:
About `object.read()`

If you have a lot of user, may be you don't wanna load every of users just few, e.g. filtered by name. Could we add `lazy: bool` parameter (by default `False`), which will be determine behavior?